test-data/unit/check-parameter-specification.test

```diff
@@ -83,7 +83,7 @@ def changes_return_type_to_str(x: Callable[P, int]) -> Callable[P, str]: ...
 def returns_int(a: str, b: bool) -> int: ...
 
 reveal_type(changes_return_type_to_str(returns_int))  # N: Revealed type is "def (a: builtins.str, b: builtins.bool) -> builtins.str"
-[builtins fixtures/tuple.pyi]
+[builtins fixtures/paramspec.pyi]
```

Comment:
Just wondering, any particular reason not to use the new fixture for all ParamSpec tests? I noticed that a few where missing. Especially between L90 - L550.

Reply:
I used the new fixture in all the tests that broke with the changes made in this PR ;) They mostly broke because `builtins.dict` is undefined in `tuple.pyi`, but we need it for a lot of `ParamSpec`-related tests if `ParamSpec.kwargs` is bound to `dict[str, Any]`.

There's no particular reason _not_ to use them in all `ParamSpec` tests -- but I wanted to keep the diff as small as possible, and the remaining tests don't need all the classes included in `paramspec.pyi`. (I think it's best to use as minimal a fixture as minimal as possible, in most situations, to avoid slowing down the test suite.)